package.json

```diff
@@ -62,6 +62,12 @@
                 "title": "%azureResourceGroups.createResourceGroup%",
                 "category": "Azure Resource Groups"
             },
+            {
+                "command": "azureResourceGroups.installExtension",
```

Comment:
I'd configure this one to never show in the command palette, since it will not work if executed that way.